### ISSUE TITLE
[Announce] VoiceOver announces incomplete changes in some situations

### ISSUE THREAD

**Issue by chaance** [radix-ui/primitives#167]
Per https://github.com/modulz/interop-ui/issues/133#issuecomment-681853914:

> Seems to work pretty well although didn't announce the full sentence in Safari + Voiceover

The closest issue we could find elsewhere suggests this is a Safari bug. https://core.trac.wordpress.org/ticket/36853

I tried implementing a similar empty string fix suggested in the WordPress bug, but that doesn't work as intended in NVDA when working with stacked notifications (as a toast likely will). This may be something we can do separately in a `Status` package where notifications are replaced completely instead of stacked, as that it primarily where the problem lies and is most likely to occur.

Referenced in https://github.com/modulz/interop-ui/pull/166

**Comment by benoitgrelard** [comment]
This primitive will be deprecated soon as it doesn't fit with our high-level use case approach.
